components/automate-chef-io/README.md

```diff
@@ -124,15 +124,15 @@ http://localhost:1313/docs/api
 
 ### Building API Docs
 
-Chef Automate  generates API documentation from protoc files comments using
+Chef Automate generates API documentation from proto files comments using
 
 * [grpc-gateway protoc-gen-swagger](https://github.com/grpc-ecosystem/grpc-gateway/tree/master/protoc-gen-swagger) as the protoc compiler
 * [OpenAPI 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) as the API schema
 * [ReDoc](https://github.com/Redocly/redoc) as the front-end display
 
 To build the API documentation during development:
 
-1. Make any edits to the .proto files necessary.
+1. Make any edits to the .proto files necessary. Note that an "Authorization Action:" section will be added automatically in the next step.
```

Comment:
Should this get mentioned in the next step?

Reply:
Great question! My reason for attaching to this step is that this is where the engineer is editing and, since most of our engineers have a pretty keen eye, they would likely have noticed that annotation in other files and naturally would just copy that format. That's OK (because if it is manually put in there the generator will see it and not add it again) but the point here is that the engineer would see that they do not need to waste time doing that as it will be done for them. (Sorry for rambling.)